src/common/utils.js

```diff
@@ -27,6 +27,7 @@ function toRippledAmount(amount: Amount): RippledAmount {
   if (amount.currency === 'XRP') {
     return xrpToDrops(amount.value)
   }
+  // $FlowFixMe: amount.issuer is not a Amount type property. Safe to remove?
```

Comment:
Good catch. `issuer` is actually the term that we use in rippled. We should add `issuer` to the `Amount` type, and keep using it here. (No hurry, but in a future release, we may deprecate `counterparty`.)

Reply:
(There are other areas in the code that will need to be updated to accommodate this)

Reply:
Say more about the other areas that need to be changed? They'll need to accommodate by  setting `issuer`? Read from `issuer`?

Reply:
@FredKSchott Right. For example, here are places where I think we'll need to [set issuer](https://github.com/ripple/ripple-lib/blob/develop/src/ledger/parse/amount.js#L17) or [read from issuer](https://github.com/ripple/ripple-lib/blob/develop/src/ledger/orderbook.js#L53). We can do this in a later PR though.